aws/scripts/sh/sanitize_branch.sh

```
@@ -1,12 +1,12 @@
-#!/bin/bash
+#!/bin/sh
 set -e
 
 # Convert BRANCH_NAME to lowercase and remove invalid characters
 sanitized_branch=$(echo "$BRANCH_NAME" | tr '[:upper:]' '[:lower:]' | sed 's/[^a-z0-9.-]//g' | sed -E 's/^[.-]+|[.-]+$//g')
 
-# If the sanitized branch name is longer than 60 characters, truncate it
-if [[ ${#sanitized_branch} -gt 60 ]]; then
-  sanitized_branch="${sanitized_branch:0:60}"
+# If the sanitized branch name is longer than 47 characters, truncate it
+if [ $(echo -n "$sanitized_branch" | wc -c) -gt 47 ]; then
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)